redisinsight/electron/util.ts

```diff
@@ -0,0 +1,13 @@
+/* eslint import/prefer-default-export: off */
+import { URL } from 'url';
+import path from 'path';
+
+export function resolveHtmlPath(htmlFileName: string) {
+  if (process.env.NODE_ENV === 'development') {
```

Comment:
I really do not understand why somewhere we are using envVars somewhere process.env and what was the goal